invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemMetadata/ItemMetadata.js

```diff
@@ -16,15 +16,15 @@ export default class ItemMetadata extends Component {
     const header = (
       <Grid.Row>
         <Grid.Column width={10} verticalAlign={'middle'}>
-          <Header as="h1">Item - {item.barcode}</Header>
+          <Header as="h1">Item - {item.metadata.barcode}</Header>
         </Grid.Column>
         <Grid.Column width={6} textAlign={'right'}>
           <CreateNewLoanModal
             itemPid={`${item.item_pid}`}
-            itemBarcode={`${item.barcode}`}
+            itemBarcode={`${item.metadata.barcode}`}
```

Comment:
Referring to my previous comment, despite the seriliazation, we still access the properties through `metadata`